pkg/client/request.go

```diff
@@ -483,7 +483,8 @@ func (r *Request) Do() Result {
 			continue
 		}
 
-		if resp.StatusCode == http.StatusServiceUnavailable {
+		// Check to see if we got a 429 Too Many Requests response code.
+		if resp.StatusCode == 429 {
```

Comment:
We should create constants somewhere for any not defined in net.http.

Here's another:
https://github.com/GoogleCloudPlatform/kubernetes/blob/master/pkg/apiserver/apiserver.go#L53

Maybe pkg/api/errors/errors.go would be a better place?